Sources/Testing/ABI/EntryPoints/EntryPoint.swift

```diff
@@ -278,6 +285,13 @@ public struct __CommandLineArguments_v0: Sendable {
 
   /// The value of the `--experimental-attachments-path` argument.
   public var experimentalAttachmentsPath: String?
+
+  /// The value(s) of the `--enable-experimental-feature` argument(s),
+  /// indicating the name(s) of experimental features to enable.
+  ///
+  /// The value of this property is not yet populated by parsing command-line
+  /// arguments, but may be in the future.
+  public var experimentalFeatures: [String]?
```

Comment:
This is a broader feature that's not directly related to this PR. I think we should leave it out for now and plan to talk to the SwiftPM folks about building this feature formally (especially if we think we're going to use the same terminology, which I think is a good idea in general.)

Reply:
Yeah, I included it to offer a practical example of how I think it would be beneficial to formalize the concept of a "feature" within the testing library, but I expected to revert that and simplify things since it's out of scope here. I'll follow up with this idea in a separate PR.